backend/note_taking/migrations/0006_block.py

```
@@ -0,0 +1,24 @@
+    operations = [
+        migrations.CreateModel(
+            name='Block',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('block_type', models.CharField(max_length=50)),
+                ('block_content', models.JSONField(default=dict)),
+                ('block_order', models.IntegerField()),
+                ('note', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='blocks', to='note_taking.note')),
+            ],
+        ),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)